submissions/damilolaSalami/domManipulation/contactForm.js

```diff
@@ -0,0 +1,35 @@
+const contactForm = document.getElementById("contactForm"),
+    fullName = document.getElementById("fullName"),
+    email = document.getElementById("email"),
+    subject = document.getElementById("subject"),
+    message = document.getElementById("message");
+
+const nameval =  /^[a-zA-Z ]+$/,
```

Comment:
@queensaldam You can attach these variables to the above and use camelCase for your variable names.  Well done!

Reply:
> @queensaldam You can attach these variables to the above and use camelCase for your variable names. Well done!

Please are you saying I should to just change the variable names to camelCase?